README.md

```diff
@@ -9,6 +9,8 @@ For a guide how to get started with this Docker image please take a look at our
 
 ## Versions
 
+[Docker Hub](https://hub.docker.com/r/owncloud/appliance/tags)
```

Comment:
So far the readme content is consistent across the docker repos. We link nowhere directly to that. Beside that we got the list below.

Reply:
Why not link?